Configuration/Skimming/python/PbPb_ZMuSkimMuonDPG_cff.py

```diff
@@ -96,8 +103,8 @@
 # build Z-> MuMu candidates
 dimuonsForPbPbZMuSkim = cms.EDProducer("CandViewShallowCloneCombiner",
                          checkCharge = cms.bool(False),
-                         cut = cms.string('(mass > 60) &&  (charge=0)'),       
-                         decay = cms.string("tightMuonsForPbPbZMuSkim looseIsoMuonsForPbPbZMuSkim")
+                         cut = cms.string('(mass > 60)'),       
+                         decay = cms.string("tightMuonsForPbPbZMuSkim looseMuonsForPbPbZMuSkim")
```

Comment:
@flodamas it appears that this change made the Tier0 replay with `CMSSW_14_1_4_patch4` fail with the following error:

```bash
2024-11-01 22:25:10,703:CRITICAL:CMSSW:Error message: An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 374951 lumi: 31 event: 43921116 stream: 2
   [1] Running path 'ZMuSkimPathPbPb'
   [2] Calling method for module CandViewShallowCloneCombiner/'dimuonsForPbPbZMuSkim'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for a container with elements of type: reco::Candidate
Looking for module label: looseMuonsForPbPbZMuSkim
Looking for productInstanceName: 

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "TryToContinue = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.
```

`looseMuonsForPbPbZMuSkim` produces a skimmed `reco::TrackCollection` while `CandViewShallowCloneCombiner` necessitates of a collection of `reco::Candidate`-s in input. 
Shouldn't this be `ConcretelooseMuonsForPbPbZMuSkim` instead?

Reply:
Hi, indeed it should be and I have already made a PR #46584 (and #46585 for backport) to fix it.

Sorry for not having caught it before!